src/vars/foreman.yml

```diff
@@ -0,0 +1,3 @@
+---
+foreman_admin_username: admin
+foreman_admin_password: "{{ lookup('ansible.builtin.password', '/tmp/passwordfile', chars=['ascii_letters']) }}"
```

Comment:
We should not store the password in `/tmp`.

We could expose https://github.com/theforeman/obsah/blob/ef8e816ea27c5a36368011e89ffe89f00b20ce16/obsah/__init__.py#L251-L256 in an variable and let people write there?

Reply:
1.4.0 now contains `obsah_state_path`